terraform/module-security-group/main.tf

```diff
@@ -0,0 +1,16 @@
+resource "aws_security_group" "payment_reconciler_sg" {
```

Comment:
We don't need to `_sg ` suffix as we already have the context of `aws_security_group `